dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -368,46 +368,39 @@ protected static StringBuilder startAuditJson(String hostName, String wlpUserDir
 
     protected static JSONObjectBuilder startAuditJsonFields(String hostName, String wlpUserDir, String serverName) {
         JSONObjectBuilder jsonBuilder = new JSONObject.JSONObjectBuilder();
-        String tempStartFields = startAuditJsonFields;
-        // We're assuming tempStartFields will never be null - i.e. updateFieldMappings is always called before this method is called
-        jsonBuilder.addPreformatted(tempStartFields);
+        // We're assuming startAuditJsonFields will never be null - i.e. updateFieldMappings is always called before this method is called
```

Comment:
just noticed... you don't need hostName / wlpUserDir / serverName in the signature of these methods anymore